chat-plugins/credits.js

```diff
@@ -406,7 +407,7 @@ exports.commands = {
 						if (err) return console.log("credits4: " + err);
 						let averageCredits = rows[0]['AVG(credits)'];
 
-						this.sendReplyBox("The richest user is currently <b><font color=#24678d>" + Tools.escapeHTML(richestUser) + "</font></b> with <b><font color=#24678d>" +
+						this.sendReplyBox("The richest user is currently <b><font color=#24678d>" + Wisp.nameColor(richestUser, true) + "</font></b> with <b><font color=#24678d>" +
```

Comment:
um what part of you read this line and was like "yea,  this looks good." 

because we need to fix that part of you before we address this pull request